etna/search/views.py

```diff
@@ -284,6 +293,28 @@ def get_meta_title(self) -> str:
             title += ' for "' + query.replace('"', "'") + '"'
         return title
 
+    def get_gtm_content_group(self) -> str:
+        return "Search"
+
+    def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
+        data = super().get_datalayer_data(request)
+        data["customDimension3"] = self.__class__.__name__
+        try:
+            data["customDimension8"] = (
+                self.title_base + ": " + self.form.cleaned_data["group"]
+            )
+        except Exception:
+            data["customDimension8"] = self.title_base + ": " + "none"
+        if self.form.cleaned_data.get("q", ""):
+            data["customDimension9"] = self.form.cleaned_data.get("q", "")
+        else:
+            data["customDimension9"] = "*"
```

Comment:
Does it return blank ""? When do we need a blank  "" ? 
Will this do instead `data.update(customDimension9= self.form.cleaned_data.get("q") or "*")`